public/js/app.js

```diff
@@ -49,20 +52,29 @@ const renderRecentlyPlayed = songs => {
   TODO - 
     Proper error handling
 */
-const handleError = err => {
-  console.log(err);
+const handleNowPlayingError = err => {
+  nowplaying.innerHTML = `
+    <div class="now-playing-container">
+      <div class="now-playing-info">
+        <div class="now-playing-title">No Music is playing</div>
+      </div>
+    </div>
+    <div class="now-playing-img">
+      <img src="./public/images/generic-image.png" alt="Generic Cover Image" />
+    </div>
+  `;
 };
 
 const displaySongs = e => {
   fetch(nowPlayingEndpoint)
     .then(response => response.json())
```

Comment:
Check here is the response has a 404 status, if it does display the no music message and album cover

Reply:
If the response is 404, wouldn't that enter that catch block? So I can check to see if the status is 404 in the catch block and render the no music message?